### PR TITLE
Search button

### PR DESCRIPTION
**Pull request by nitanagdeote**
Search button width.

### CODE REVIEW

File: packages/neoFrontend/src/NavBar/Search/styles.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import styled from 'styled-components';
 
 export const SearchInput = styled.input`
-  width: 100%;
+  width: 120%;
```
Comment (stushurik) [inline packages/neoFrontend/src/NavBar/Search/styles.js:4]: i think this is bad way

Comment (stushurik) [reply packages/neoFrontend/src/NavBar/Search/styles.js:4]: 120% would really different on devices

Comment (curran) [reply packages/neoFrontend/src/NavBar/Search/styles.js:4]: Indeed. A bit risky.

I was searching the other day for where the width of this box comes from - could not figure it out.

**Comment by curran** [comment]
Closing as the approach is risky for mobile. Open to the idea, though, of increasing the width of that search box.

File: packages/neoFrontend/src/NavBar/Search/styles.js (hunk 1 above)
Comment (nitanagdeote) [reply packages/neoFrontend/src/NavBar/Search/styles.js:4]: > i think this is bad way

You are right.

**Pull request closed without merging**